omega/omega-connector/omega-connector-grpc/src/test/java/org/apache/servicecomb/saga/omega/connector/grpc/LoadBalancedClusterMessageSenderTest.java

```diff
@@ -278,6 +278,10 @@ public Boolean call() throws Exception {
     assertThat(eventsMap.get(8080).isEmpty(), is(true));
     assertThat(eventsMap.get(8090).isEmpty(), is(true));
 
+    //it seems in Windows environment we need wait a short time in order to make sure onError in GrpcCompensateStreamObserver triggered
+    // after servers shutdown before restart server
+    Thread.sleep(2000);
```

Comment:
It's better to wait the server terminate instead of just sleep here.

Reply:
`server.awaitTermination()` not work